java/compaction/compaction-job-execution/docker/Dockerfile

```diff
@@ -11,7 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM amazoncorretto:11
+FROM ubuntu:22.04
+
+RUN apt-get update
+RUN apt-get install -y wget gnupg
+RUN wget -O- https://apt.corretto.aws/corretto.key | apt-key add -
+RUN echo 'deb https://apt.corretto.aws stable main' | tee /etc/apt/sources.list.d/corretto.list
+RUN apt-get update
+RUN apt-get install -y java-11-amazon-corretto-jdk
```

Comment:
The apt-get update and install commands could be combined and have a `rm -rf /var/lib/apt/lists/*` at the end. This would avoid extra apt cache files in the Docker image.

Why change to Ubuntu over Amazon Linux?

Reply:
I think it was the Amazon Corretto 11 Docker image has too old of a glibc version on it. So I've migrated to Amazon Linux 2023 with Corretto 11 installed from a yum repository.